deploy/add_combine_field_values_function.psql

```diff
@@ -0,0 +1,29 @@
+-- Deploy afya-tek:add_combine_field_values_function to pg
+-- requires: setup
+-- requires: initial_migrations
+
+BEGIN;
+
+-- lets make it possible to create this function in any schema provided by the
+-- :"schema" variable - and default to public.
+SET search_path TO "schema",public;
+
+CREATE OR REPLACE FUNCTION combine_field_values(form_data_hr TEXT, form_data_raw TEXT) RETURNS VARCHAR AS $$
```

Comment:
Let's add a comment describing what this is for.

Reply:
> Let's add a comment describing what this is for.

This is done.